components/modules/rmt.c

```diff
@@ -155,6 +162,13 @@ static int lrmt_rxsetup(lua_State *L) {
       config.rx_config.idle_threshold = threshold;
     }
     lua_pop(L, 1);
+
+    lua_getfield(L, 3, "slots");
+    int slots = lua_tointeger(L, -1);
```

Comment:
Same comment as above - this doesn't look like slots is optional?

Reply:
lua_tointeger returns 0 if the value cannot be converted to an integer and this is checked in the next line